packages/@adobe/spectrum-css-temp/components/colorhandle/skin.css

```diff
@@ -34,3 +34,17 @@
 .spectrum-ColorHandle-color {
   box-shadow: inset 0 0 0 var(--spectrum-colorhandle-outer-border-size) var(--spectrum-colorhandle-outer-border-color);
 }
+
+@media (forced-colors: active) {
+  .spectrum-ColorHandle {
+    forced-color-adjust: none;
+    --spectrum-colorhandle-inner-border-color-disabled: GrayText;
+    --spectrum-colorhandle-fill-color-disabled: GrayText;
+    --spectrum-colorhandle-inner-border-color: ButtonText;
+    --spectrum-colorhandle-outer-border-color: ButtonFace;
+    &.is-focused {
+      border-color: Highlight;
+    }
```

Comment:
Should focused border color remain ButtonText, and perhaps make the border slightly thicker, like you do in some places where you add an `outline` style?

```css
@media (forced-colors: active) {
  .spectrum-ColorHandle.is-focused {
    border-width: calc(var(--spectrum-colorhandle-inner-border-size) * 1.5);
  }
}
```

Reply:
I mean it could stay ButtonText at the same width and the fact that the handle gets significantly bigger is enough. The fact that there are inconsistencies between the focus style from one component to the next is a reality of implementing high contrast mode due to the very limited colour palette. I don't propose to remove all such inconsistencies. 

I kind of wish I had put them in a different PR as they are the most problematic IMO.